src/protos/controls.proto

```diff
@@ -0,0 +1,28 @@
+syntax = "proto3";
+
+/*************************************************/
+/******** Main messages that will be sent ********/
+/*************************************************/
+
+message controlConfig {
+    // The main firing trigger that requests a dart out the barrel, ASAP
+    int32 triggerSwitchPin = 1;
+    SwitchOrientation TriggerSwitchOrientation = 2;
+    // An optional switch that requests the flywheels rev up to full (or idle? partial?) speed
+    int32 optRevSwitchPin = 3;
+    SwitchOrientation optRevSwitchOrientation = 4;
+    // Which button is this?
+    int32 buttonPin = 5;
+    // This is internally routed on Detlaff; you probably don't want to touch it
+    // Is there any reason a user should want to touch this?
```

Comment:
batteryAdcPin is determined by the hardware on the board so it's not something that the user would need to change, but it might, and I believe already has, changed with different board revisions. So if we want to have a single binary file that you can flash to any Dettlaff version and then configure the parameters, then things like batteryAdcPin would have to be a parameter. It would save us from having to generate separate binary files for each board revision on every firmware release, and I think we have enough performance headroom for it to be ok? buttonPin though always has to be pin 0 because that's the pin that puts the ESP32 into firmware upload mode.